docs/main-entrypoint.md

```diff
@@ -15,5 +15,44 @@ module.exports = async function service(fastify) {
   })
 }
 ```
+As you can see the function must be declared `async` and must be exported as the root of the module.
 
-As you can see the function must be declared `async` and must be exported as the root of the module.  
+## Custom Status Routes
+With `lc39` your service will automatically inherit **two** fixed routes for getting infomation on the
+service:
+
+- `GET /-/healthz`
+- `GET /-/ready`
+
+The first route can be used as a probe for load balancers, status dashboards
+and as a [`helthinessProbe`][k8s-deployment-probes] for [Kubernetes][k8s].  
+By default, the route will always response with an `OK` status and the `200` `HTTP` code as soon as the service is up.
+
+The second route can be used as a [`readinessProbe`][k8s-deployment-probes] for Kubernetes.  
+As the first route, the default implementation of this endpoint will always respond
+`OK` status and the `200` `HTTP` code as soon as the service is up.
+
+The default implementations are a nice placeholder until you can add some logic tied to your service.  
+For doing so you can add two new `module.exports` to your main entrypoint that will be used instead of the defaults.
+
+```javascript
+module.exports.readinessHandler = function readinessHandler(request, reply) {
+  // Add your custom logic for /-/ready here
+}
+module.exports.healthinessHandler = function readinessHandler(request, reply) {
```

Comment:
The name of the function should be `healthinessHandler` instead of `readinessHandler`. Correct?